pkg/game/board.go

```diff
@@ -626,16 +626,18 @@ func (board *board) ScoreRoads(placedTile elements.PlacedTile, forceScore bool)
 	return scoreReport
 }
 
-func (board *board) ScoreFinalMeeples() elements.ScoreReport {
+func (board *board) ScoreMeeples(final bool) elements.ScoreReport {
```

Comment:
`ScoreNotFinalMeeples()` called `MeepleInReport()` to check if the meeple is in the score report already - I believe this is still needed in some cases. Since the tests pass despite it, could you add a test case where the lack of this check is an issue for mid-game scoring?

Reply:
made new test for that